Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -321,6 +322,7 @@ extension Scheme {
     public init(name: String, target: Target, targetScheme: TargetScheme, project: Project, debugConfig: String, releaseConfig: String) {
         self.init(
             name: name,
+            wasCreatedForAppExtension: targetScheme.wasCreatedForAppExtension,
```

Comment:
The tests are failing because `wasCreatedForAppExtension = "NO"` is being added to the scheme where it's not expected. Xcode doesn't write this if it's false. This could be classified as a bug in XcodeProj, but to fix it here let's just use nil for false
```suggestion
            wasCreatedForAppExtension: targetScheme.wasCreatedForAppExtension ? targetScheme.wasCreatedForAppExtension : nil,
```

Reply:
ahh okayy. thanks